app/assets/stylesheets/nav.scss

```diff
@@ -44,6 +44,10 @@ body.home {
   margin-right: 5%;
 }
 
+.flash {
+  color: orange;
```

Comment:
Color `orange` should be written in hexadecimal form as `#ffa500`<br>Color literals like `orange` should only be used in variable declarations; they should be referred to via variable everywhere else.